snippets/xen_dom0/snippet.yml

```diff
@@ -0,0 +1,9 @@
+name: xen_dom0
+append:
+  EXTRA_DTC_OVERLAY_FILE: xen_dom0.overlay
+  EXTRA_CONF_FILE: xen_dom0.conf
+
+boards:
+  qemu_cortex_a53:
```

Comment:
you can add `rcar_h3ulcb_ca57` and `salvator_xs_m3` too, support of them have been added to mainline recently

Reply:
@xakep-amatop, added in separate commit.